raiden/network/rpc/client.py

```diff
@@ -55,6 +56,7 @@
 
 log = slogging.getLogger(__name__)  # pylint: disable=invalid-name
 solidity = _solidity.get_solidity()  # pylint: disable=invalid-name
+client_settings = {'block_gas_limit': GAS_LIMIT}
```

Comment:
Please, do not use globals, we run multiple instances on the same process on our tests and this can lead to race conditions.